superset/db_engine_specs/mysql.py

```diff
@@ -93,6 +104,21 @@ class MySQLEngineSpec(BaseEngineSpec):
 
     type_code_map: Dict[int, str] = {}  # loaded from get_datatype only if needed
 
+    custom_errors = {
+        INVALID_ACCESS_REGEX: (
+            __('Either the username "%(username)s" or the password is incorrect.'),
+            SupersetErrorType.TEST_CONNECTION_ACCESS_DENIED_ERROR,
```

Comment:
Let's rename the regex to `ACCESS_DENIED_REGEX`, to be consistent with the other errors.